### ISSUE TITLE
The user isn't provided an install location on install

### ISSUE THREAD

**Issue by notnsane** [Squirrel/Squirrel.Windows#860]
I swear I did three searches on both closed and opened issues but couldn't find anything very related, so I'm submitting the issue just in case, also I want to provide a couple user cases on why this should be (optionally) provided to the user on install:

1. It's how software is being deployed since decades ago (refers to the mental model of the installation process) - though creating user registry entries currently is not ideal (due to UWP apps rejecting that), so the installer should clean up itself after used.
2. With the advent of SSD drives, many users are switching to them, and those are way smaller than HDDs. That being the case, maybe many users will decide to have only _very_ essential apps on a C:\ partition and some heavy duty apps like games or multimedia apps. Those will get filled fast, and moving non-essentials apps (maybe the ones generated by squirrel) is a good idea, generally.
3. In some cases, a user just want to install their apps on other partitions than C:\ to keep their folders and searches tidy.

tl;dr forcing installs to C:\ is a bad practice **in the long run**

**Comment by paulcbetts** [comment]
Yep, Squirrel will never do this, check the README for details (specifically the "No Wizards" part)
